katello/katello-backup

```diff
@@ -44,14 +44,22 @@ optparse = OptionParser.new do |opts|
   @dir = ARGV[0].dup
 end
 
+def run_cmd(command)
+  `#{command}`
+  if $?.exitstatus != 0
```

Comment:
The config file backup will generally return exitstatus 2, as the file systems are different for puppet 3 vs puppet 4, which means some files that are called in the compression don't exist, based on which puppet is being used.